.github/workflows/ci.yml

```diff
@@ -45,6 +54,7 @@ jobs:
         run: rush build -t @azure/communication-react
       # Verify no uncommitted api extractor changes
       - name: API Extractor Check
+        if: ${{ matrix.flavor != 'stable' }}
```

Comment:
We should separately generate API files for the two flavors of builds (to two different folders), and then check there are no uncommitted changes in both cases.

Please add a TODO here if you intend to do that once you've worked on generating the API review files separately.

Reply:
This could be a seperate PR but +1 to this - it would be very useful when introducing a beta feature to see the API change _only_ in the beta file and no nothing crept into the stable API